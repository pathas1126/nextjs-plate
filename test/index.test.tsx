import { render } from "@testing-library/react";
import Index from "../pages/index";

describe("Index", () => {
  it("renders Index without crashing", () => {
    const utils = render(<Index />);
    expect(utils.container).toMatchSnapshot();
  });
  it("render Hello World", () => {
    const { getByText } = render(<Index />);
    getByText("Hello Next.js ðŸ‘‹");
  });
});
