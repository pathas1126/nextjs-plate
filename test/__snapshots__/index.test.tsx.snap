// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders Index without crashing 1`] = `
<div>
  <div>
    <header>
      <nav />
    </header>
    <h1>
      Hello Next.js ðŸ‘‹
    </h1>
    <footer>
      <hr />
      <span>
        I'm here to stay (Footer)
      </span>
    </footer>
  </div>
</div>
`;
